pipelines/migration/projeto_subsidio_sppo/flows.py

```
@@ -295,7 +295,7 @@
             key=unmapped("subsidio_data_check"),
         )
 
-        missing_timestamps = task(lambda s: True if len(s) > 0 else None)(timestamps)
+        missing_timestamps = task(lambda s: True if len(s) > 0 else False)(timestamps)
```

Ruff (0.12.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)